pages/posts/w3c-wot-case-study.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "How the W3C Web of Things brings JSON Schema to the Internet of Things"
+date: 2022-08-17T06:00:00+00:00
+type: Case Study
+cover: /img/posts/2022/w3c-wot-case-study/cover.webp
+authors:
+  - name: Juan Cruz Viotti
+    photo: /img/avatars/jviotti.webp
+    link: https://twitter.com/jviottidc
+excerpt: "Using JSON Schema at the W3C Web of Things to create an interoperability layer so that different IoT platforms, protocols and standards can operate together"
+language: en
+---
+
+## Challenge
+
+The Internet of Things (IoT) is the network of physical "smart" objects that
+exchange data with other devices over the Internet. While communication and
+interoperability are by definition the crux of the Internet of Things, the
+emergence of custom or proprietary solutions results in devices that cannot
+talk to each other due to differences in data interchange mechanisms.
+
+To integrate these disparate devices, developers must work with a growing set
+of protocols, serialization formats and API specifications. This results in
+repetitive, non-scalable and error-prone work that is difficult to automate.
+
+While technologies like OpenAPI and AsyncAPI largely solve this problem in the
+context of Web APIs, they fall short for describing networks of non-HTTP
+multi-protocol devices and do not consider different modes of interaction based
+on their meaning in the physical world.
+
+## Solution
+
+To solve these problems, the W3C Web of Things works on providing standardized
+building blocks that make use of JSON Schema.
+
+JSON Schema is used for validating descriptions of network-facing capabilities
+of physical entities called [Thing Descriptions][thing-description], and to
+model and describe data sent by Internet of Things consumers and producers in a
+multi-protocol manner.
+
+The W3C Web of Things specifications are on board with JSON Schema since Draft
+4, and even the first draft versions already catered to data models of Internet
+of Things devices.  "Currently, we are on JSON Schema Draft 7 and expect to
+move to JSON Schema
+[2020-12](https://json-schema.org/specification-links.html#understanding-draft-names-and-numbers)
+or a newer one when starting our new charter in February 2023," said Ege
+Korkan, Research Scientist at Siemens and W3C Specifications editor.
+
+"We observe JSON Schema becoming more and more stable over the years and
+Postman's support to the community gives us even more confidence on it,"
+continued Ege Korkan.
+
+JSON Schema is delivering exciting features, and more importantly these
+features are becoming more and more consistent across implementations.  For
+example, Ege Korkan added, "vocabularies are very promising and we plan to
```

Comment:
```suggestion
example, Ege Korkan added, "JSON Schema vocabularies are very promising and we plan to
```

better to be specific since vocabulary is a very commonly used term in WoT and Semantic Web. If you prefer another way to say that, I would be also fine :)